### PR TITLE
195 Add locale variable to the Simplified Login widget

### PR DESCRIPTION
**Pull request by zamorastp**
See also: https://github.schibsted.io/spt-identity/simplified-login-widget-frontend/pull/58

### CODE REVIEW

File: src/identity.d.ts [hunk 1]
```diff
@@ -572,6 +572,11 @@ export type SimplifiedLoginWidgetOptions = {
      * - expected encoding of simplified login widget. Could be utf-8 (default), iso-8859-1 or iso-8859-15
      */
     encoding: string;
+    /**
+     * - expected locale of simplified login widget. Should be provided in a short format like 'nb',
+     * 'sv'. If not set, a value from the env variable is used.
+     */
+    locale?: "nb"|"sv"|"fi"|"da";
```
Comment (saltwoo) [inline src/identity.d.ts:579]: `en` should be supported as well

**Review by filipprasalek — COMMENTED** [review]
It seems that you have downgraded package-lock.json to version 1. It could have happened due to old version of NPM and/or Node, used to install dependencies. Rerunning `npm install` with NPM > v7 should do the trick.

**Comment by zamorastp** [comment]
> It seems that you have downgraded package-lock.json to version 1. It could have happened due to old version of NPM and/or Node, used to install dependencies. Rerunning `npm install` with NPM > v7 should do the trick.

Thank. I will fix that :)

**Review by filipprasalek — APPROVED** [review]
LGTM

**Pull request merged**